File: JS-Basics/First Steps/Exercise/suppliesForSchool.js
function suppliesForSchool(input) {
    let pensPrice = 5.8;
    let markersPrice = 7.2;
    let detergent = 1.2;

    let pensAmount = Number(input[0]);
    let markersAmount = Number(input[1]);
    let detergentQuantity = Number(input[2]);
    let discount = Number(input[3]) / 100;

    let totalPensPrice = pensPrice * pensAmount;
    let totalMarkersPrice = markersPrice * markersAmount;
    let totalDetergent = detergent * detergentQuantity;

    let price = totalPensPrice + totalMarkersPrice + totalDetergent;
    let totalPrice = price - (price * discount);

    console.log(totalPrice);
}

suppliesForSchool(["2", "3", "4", "25"]);

File: Python/SoftUni PB/Exercises/Python Basics/First Steps in Coding - Exercise/Supplies for school.py
pen_package = 5.8
marker_package = 7.2
detergent_liter = 1.2
pens = int(input())
markers = int(input())
detergent = int(input())
discount = int(input())
price = (pens * pen_package) + (markers * marker_package) + (detergent * detergent_liter)
total = price - (price * (discount / 100))
print(total)
